tests/rpc/generate_mainnet_tester_chain.py

```diff
@@ -0,0 +1,126 @@
+import os
+import time
+
+from eth_keys import (
```

Comment:
you can just do `from eth_keys import keys` and it'll automatically *just* work with whatever the configured backend is.  (`keys` will be an instance of `KeyAPI`)

Reply:
TBH, this is copy-pasta from https://github.com/ethereum/eth-tester/blob/168c7a59/eth_tester/backends/pyevm/main.py

I'll take another pass through it, but since it was just for generating the test database, I wasn't holding it to the same standard as prod code.